scipy/optimize/tests/test_constraints.py

```diff
@@ -76,12 +76,23 @@ def test_prepare_constraint_infeasible_x0():
     bounds = Bounds(lb, ub, enforce_feasibility)
     pytest.raises(ValueError, PreparedConstraint, bounds, x0)
 
+    # check is_satisifed method
+    pc = PreparedConstraint(Bounds(lb, ub), [1, 2, 3])
+    assert(not pc.is_satisifed([1, 2, 3]))
```

Comment:
nitpick: would be better to use `assert` as a keyword

Reply:
I'm not sure what you mean by using `assert` as a keyword.

Reply:
not as a function call but `assert not pc.is_satisifed([1, 2, 3])`

Reply:
Done, good catch

Reply:
Looks like all `assert` statements in the diff now -- marking as resolved